clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/op/handler/KubernetesDeploymentHandler.java

```diff
@@ -79,7 +79,7 @@ public Status status(KubernetesManifest manifest) {
     if (manifest.getApiVersion().equals(EXTENSIONS_V1BETA1)
         || manifest.getApiVersion().equals(APPS_V1BETA1)
         || manifest.getApiVersion().equals(APPS_V1BETA2)) {
-      if (!manifest.isNewerThanObservedGeneration()) {
+      if (manifest.isNewerThanObservedGeneration()) {
```

Comment:
Hmm. What's the logic behind this change? Was the previous logic broken?

Reply:
It wasn't broken, but inverted. The method name states `isNewerThanObservedGeneration` but the original code in this method would do just the opposite, returning false if `generation > observedGeneration`, and all the usages were negated. I felt like this was a reasonable refactor.

Reply:
Ah right. Agreed!